### PR TITLE
Filters

### PR DESCRIPTION
**Pull request by MarkVed17**
- Created & Consumed Filters Context
- Filters Functionalities

Link:
https://deploy-preview-4--camp-cretaceous.netlify.app/

Preview:

recording

### CODE REVIEW

**Review by ark2002 — APPROVED** [review]
the filters work fine
- file structure looks good and systematic.

File: src/frontend/components/Filters/Filters.css [hunk 1]
```diff
@@ -6,7 +6,8 @@
   align-self: flex-start;
   gap: 1.5rem;
   min-width: 15rem;
-  width: 30%;
+  /* width: 30%; */
```
Comment (vijaytembugade) [inline src/frontend/components/Filters/Filters.css:9]: commented code!

**Review by vijaytembugade — APPROVED** [review]
Amazing work Vedant!

File: src/frontend/components/Filters/Filters.js [hunk 2]
```diff
@@ -1,13 +1,29 @@
 import React from "react";
+
+import {
+  CLEAR_FILTERS,
+  SORT_BY,
+  CATEGORY,
+  RATING,
+  MAX_PRICE,
+} from "../../constants/filtersConstants";
```
Comment (t007rushi) [inline src/frontend/components/Filters/Filters.js:9]: Lik that you have constants

**Review by t007rushi — APPROVED** [review]
Good to merge

**Pull request merged**